lib/serialization/timeSpanSpec.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+import { TypeSpec, createValidationErrorMessage } from "./typeSpec";
+import { isDuration, Duration, duration } from "moment";
```

Comment:
At some point we want to move away from moment or extract out duration specific parts of moment and create a separate library. We need moment only for duration and moment brings ~60KB along with it in the runtime, thus making the runtime bulky.

Reply:
I think that's a great idea. I've created [this issue](https://github.com/Azure/ms-rest-js/issues/47) for tracking that.